src/components/Comments/WriteComment.js

```diff
@@ -160,6 +181,14 @@ export const styles = StyleSheet.create({
     // zIndex: 1,
     paddingBottom: 14
   },
+  isViewOnlyColumnContainer: {
+    width: '100%',
+    display: 'flex',
+    flexDirection: 'column',
+    borderTopWidth: 1,
+    borderTopColor: COLORS.gray1,
+    position: 'absolute'
+  },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new style `isViewOnlyColumnContainer` has been added. This seems to be used when the component is in view-only mode.



<!-- This is an auto-generated comment by OSS CodeRabbit -->